steps/33/README.md

```diff
@@ -116,9 +116,43 @@ There is no meaningful difference between what we had before with 3 generic para
 
 In this context, we call the trait `Config` because it is used to configure all the types for our Pallet.
 
-### Implementing the Config Trait
+### The Power of Type-Level Abstraction
```

Comment:
I don't really see the term "type-level abstraction" show up that much in Rust.

Generics, Associated Types, etc is what I see more often.

Probably will change this to follow rust conventions.